iroh/src/metrics.rs

```diff
@@ -18,6 +29,13 @@ impl Default for Metrics {
             requests_total: Counter::new("Total number of requests received"),
             bytes_sent: Counter::new("Number of bytes streamed"),
             bytes_received: Counter::new("Number of bytes received"),
+            download_bytes_total: Counter::new("Total number of content bytes downloaded"),
+            download_time_total: Counter::new("Total time in ms spent downloading content bytes"),
```

Comment:
doesn't seem that useful to me.  a histogram of download speeds or something might be more useful.  but just removing this would also be fine i think

Reply:
This was added because @Arqu said "only do counters in metrics if you can" and @b5 wanted to have download speed stats for the demo. So I'd defer to them on what to do here :)